src/db.server.ts

```diff
@@ -0,0 +1,12 @@
+import { PrismaClient } from '@prisma/client';
+
+const globalPrismaVariable = globalThis as unknown as { prisma: PrismaClient };
+
+// TODO: use $env module instead of import.meta.env
+export const prisma =
+	globalPrismaVariable.prisma ||
+	new PrismaClient({
+		log: import.meta.env.NODE_ENV === 'development' ? ['query', 'error', 'warn'] : ['error'],
```

Comment:
Ignore the log variable for now... we'll chat about the best way to enable logging.